x-pack/plugins/transform/public/app/hooks/use_index_data.ts

```diff
@@ -61,7 +66,7 @@ export const useIndexData = (
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [JSON.stringify(query)]);
 
-  const getIndexData = async function () {
+  const fetchDataGridData = async function () {
```

Comment:
I think for this PR, the histogram button needs to be hidden for the Pivot Preview table as the charts only work where we can use ES aggs.

Reply:
Added a prop to allow hiding the button in 510f7b6